src/main/java/graphql/annotations/processor/retrievers/GraphQLFieldRetriever.java

```diff
@@ -1,17 +1,3 @@
-/**
- * Copyright 2016 Yurii Rashkovskii
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- */
```

Comment:
I don't see a point to mention the license anymore, because the founder of the library is not active anymore, and the library have been changed a lot since.
But I don't familiar with the rules, so if it's not legal/legitimate, I will put it back